tensorflow/python/util/tf_decorator_test.py

```diff
@@ -207,16 +205,15 @@ def testSetsTFDecoratorArgSpec(self):
         defaults=(1, 'hello'))
     decorated = tf_decorator.make_decorator(test_function, test_wrapper, '', '',
                                             argspec)
-    decorator = getattr(decorated, '_tf_decorator')
-    self.assertEqual(argspec, decorator.decorator_argspec)
+    self.assertEqual(argspec, decorated._tf_decorator.decorator_argspec)
```

Comment:
You'll probably get lint warnings here

Reply:
Whoops, the warnings were buried under other linter issues. Fixed in c82356c2096f911a28b94c14ff6bb055bf5361c6